ext/reflection/php_reflection.c

```diff
@@ -6108,17 +6121,20 @@ ZEND_METHOD(ReflectionProperty, getRawValue)
 
 	GET_REFLECTION_OBJECT_PTR(ref);
 
-	if (prop_get_flags(ref) & ZEND_ACC_STATIC) {
```

Comment:
Nit: I don't think this needs to change, `ref` cannot change static-ness in child classes.

Reply:
When it is private we do not actually update the property info in reflection_property_get_effective_prop, but I missed the dynamic property part.